src/main/java/Ui.java

```diff
@@ -0,0 +1,55 @@
+public class Ui {
+
+    public Ui(){
+    }
+
+    /**
+     * This method forms a divider line.
+     * @return String This is the line to be return.
+     */
+    public String line(){
+        return "____________________________________________________________\n";
+    }
+
+    /**
+     * This method prints the welcome message when user first run the program.
+     */
+    public void showWelcome(){
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?");
+        System.out.print(line());
+    }
+
+    /**
+     * This method return the Bye Message.
+     * @return String This is the bye message.
+     */
+    public String byeMessgae(){
```

Comment:
I think there is a typo here for the method name. You may want to double-check on this.